packages/features/vault/src/side-drawers/manage-vault/use-manage-vault.tsx

```diff
@@ -4,31 +4,56 @@ import { FormattedMessage } from 'react-intl';
 
 export function useManageVault() {
   const { pathname: currentPath } = useLocation();
-  const reduceLeverageOptions = [
```

Comment:
I feel like the way you have it here is cleaner...one way you can do this is actually pull in eligibleActions in this hook and then do:
```
return {
  reduceLeverageOptions: reduceLeverageOptions.filter((o) => eligibileActions.includes(o.key))
  ...
}
```

Reply:
I can change this if you like. What we get from using the big object is here is that the manage vault side drawer only needs to have one loop and one block of JSX to display all the options instead of having two that are basically the same.

Reply:
Ok up to you, I think it's fine either way